### DISCUSSION TITLE
Want to contribute but lack a roadmap

### DISCUSSION THREAD

**Discussion by shlroland** [prosekit/prosekit#683] [General]
@ocavue Hi, it's a great project. Is this considered a follow-up project to the "remirror" project? I also want to make some contributions. Is there a complete roadmap?

**Comment by ocavue** [comment]
Thanks for your kind words. I have utilized the knowledge gained from developing Remirror and other projects to create ProseKit. You can find the roadmap [here](https://github.com/orgs/prosekit/projects/2). Additionally, letting me know the features you most want prioritized would be extremely helpful.
